extensions/objects/templates/list_of_sets_of_html_strings_editor.html

```diff
@@ -0,0 +1,8 @@
+<div class="container-fluid">
+  <div ng-repeat="choice in choices track by $index">
```

Comment:
How about sorting these items according to their selected ranking? For e.g. if there are 4 items A, B, C and D displayed in that order and A has rank 2, B has rank 1 and C and D are unranked. Then change the order to B, A, C and D. So sort items according to their ranking. Also show a small animation of it moving up and down, too?

Reply:
Yes, it can be the case. But I am not sure about implementation because here we're already changing the dropdown list content dynamically. Now we need to delete one element from DOM dynamically and move to other place that too with animation. I'll take a look.

Reply:
This is still remaining. Is this not possible (without animation)?

Reply:
Ah... right now, I can't find the solution on internet. It may be possible using DOM (not sure).

Reply:
You can modify the order of items in $scope.choices, right? Then they will be printed in new order. This can be done easily by keeping track of html and rank in choices and then sorting it according to rank.

Reply:
This way the $scope.choices array will be modified and may be items rendered in html template. Can we take it later on as enhancement to this interaction?

Reply:
I dont think its very hard thing to do. $scope.choices will be modified but solution is simple.

Reply:
I dont think its very hard thing to do. $scope.choices will be modified but solution is simple.

Reply:
Looks like this isn't resolved yet.

Reply:
Done